django/src/rdwatch_scoring/api.py

```diff
@@ -0,0 +1,7 @@
+from ninja import NinjaAPI
+
+from .views import router as scores
+
+api = NinjaAPI()
+
+api.add_router('/scores', scores)
```

Comment:
The convention we've used so far is to include the trailing slash on the router prefixes:
```suggestion
api.add_router('/scores/', scores)
```